### PR TITLE
Directive for swapping column names

### PR DESCRIPTION
**Pull request by nitinmotgi**
Checklist

- [x] Documentation
- [x] Test case
- [x] Updated README.md

### CODE REVIEW

File: wrangler-core/src/test/java/co/cask/wrangler/steps/SwapTest.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    };
+
+    List<Record> records = Arrays.asList(
+      new Record("a", 1).add("c", "sample string")
+    );
+
+    records = PipelineTest.execute(directives, records);
+  }
+
+}
```
Comment (russorat) [inline wrangler-core/src/test/java/co/cask/wrangler/steps/SwapTest.java]: newline at end of file

**Comment by russorat** [comment]
newline at end of one file, otherwise LGTM

File: wrangler-core/src/test/java/co/cask/wrangler/steps/SwapTest.java (hunk 1 above)
Comment (nitinmotgi) [reply wrangler-core/src/test/java/co/cask/wrangler/steps/SwapTest.java]: Fixed

**Comment by nitinmotgi** [comment]
Merging.

**Pull request merged**